### PR TITLE
chore: update core AFJ to 0.2.2

### PR DESCRIPTION
**Pull request by amanji**
Signed-off-by: Akiff Manji <akiff.manji@gmail.com>

# Summary of Changes

Updates core AFJ module to 0.2.2 to align with app.

# Related Issues

#491 

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [ ] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [ ] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [ ] Updated documentation as needed for changed code and new or modified features;
- [ ] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: app/android/build.gradle [hunk 1]
```diff
@@ -13,7 +13,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:4.2.2")
+        classpath('com.android.tools.build:gradle:7.2.2')
```
Comment (JamesKEbert) [inline app/android/build.gradle]: The gradle update is concerning--as that likely will mess with RN until we finish updating to a newer RN version.

Comment (amanji) [reply app/android/build.gradle]: Should I just commit the NPM package update?

Comment (JamesKEbert) [reply app/android/build.gradle]: That's what I think I would do--the gradle gets updated with 0.68.0, so I think it causes some issues to upgrade now (as we tried previously for a different thing)

**Review by JamesKEbert — APPROVED** [review]
LGTM

**Pull request merged**